i18n/ads-language-pack-de/translations/extensions/debug-auto-launch.i18n.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
You have a lot of files from extensions that we don't have in our repo. I'd suggest removing those and only keeping the ones that we actually have to reduce the overall size